woocommerce/woocommerce-android
# [Customer Search] Modify navigation so after customer details confirmed order creation screen is shown

Closes: #9307


### Description

The PR adds navigation back from customer details screen right to the order creation screen. This is behind feature flag

### Testing instructions

* Start order creation
* Add customer details
* Select a customer from the list
* Notice that the search is hidden on the order details screen
* Notice that clicking on "done" leads to the order creation screen and the selected customer is bypassed

Test the same on release build - notice that old behavior still in place with search and navigation

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/312433bb-c241-42e4-ba3e-6ae398ec7413
